schedule/yast/installer_extended/installer_extended.yaml

```diff
@@ -27,7 +27,8 @@ schedule:
   - installation/system_role/select_role
   - installation/partitioning/accept_proposed_layout
   - installation/releasenotes
-  - installation/installer_timezone
+  - installation/clock_and_timezone/validate_timezone_configuration
```

Comment:
I think we should have a dedicated test suite for this to check defaults, I can see this installer_extended for doing some more actions, clicking there, open other windows, etc. Also it would need to be commented in the header of this yaml and make the other installer_extended yamls consistent, plus adding for it test data, my recommendation would be to not added here.

Reply:
ok, removed. We should revisit this `installer_extended`. Also, I do not like the name, because it says nothing about what it tests inside. Maybe we can make it more specific, because right now it is collector of the test modules that do not fit another test suites:)